spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -1181,5 +1196,26 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
       }
     }
   }
+
+
+  // this three variables are for mkldnn int8. because container maybe have more than 1 Tensor
+  // output, so use a mutable ArrayBuffer[Float] to handle the ConcatTable, Graph and so on
+  // mask == 0 means we use single scales, which will get the max of whole tensor (in/out).
+  // scalesOfInput will maintain the max value of input tensor
+  // scalesOfOutput will maintain the max value of output tensor
+  var scalesOfInput: Int8ScalesAndMask = new Int8ScalesAndMask(Int8ScalesAndMask.SINGLE_SCALE)
```

Comment:
I prefer not initializing it here as it's only used in Int8, they are null for other cases